# Topics to record or playback.  Multiple topics can be listed.
string[] record_topics

# string to prepend to each bag file. May not include directories.
string log_prefix

# the root directory used for saving bag files.
string data_location